exercises/perfect-numbers/example.rb

```diff
@@ -0,0 +1,39 @@
+class PerfectNumber
+  VERSION = 1
+  attr_accessor :num
+  
+  def initialize(num)
+    raise "#{num} is not a natural number." if num < 0
```

Comment:
Hey Bernardo. Was having an initialize and an instance variable a good idea on my part? I think if I wanted to write tests for this, I would have enforce the users to also call their instance variable num which doesn't seem correct.